### DISCUSSION TITLE
llvm-clang version

### DISCUSSION THREAD

**Discussion by garadeaniket** [intel/llvm#9548] [Q&A]
which version of llvm-clang is used in Intel(R) oneAPI DPC++/C++ Compiler 2023.0.0?

**Comment by AlexeySachkov** [comment]
Hi @garadeaniket,

The closest open-source tag/release to oneAPI DPC++/C++ Compiler 2023.0.0 is [oneAPI DPC++ Compiler 2022-09](https://github.com/intel/llvm/releases/tag/2022-09). It is made from commit 0f579bae55c48d810e1ed76db29229c854e61d5e and at that time LLVM had version 16.0.0:

https://github.com/intel/llvm/blob/0f579bae55c48d810e1ed76db29229c854e61d5e/llvm/CMakeLists.txt#L18-L26
